### PR TITLE
http: add http api for get db and table info that is related to the tableID

### PR DESCRIPTION
**Pull request by crazycs520**
### What problem does this PR solve? 
add http api to get db and table info that is related to the tableID.
This is for the flash use.
```shell
curl http://{TiDBIP}:10080/db-table/{tableID}
```
This will return below now:
```go
type dbTableInfo struct {
	DBInfo        *model.DBInfo    `json:"db_info"`
	TableInfo     *model.TableInfo `json:"table_info"`
	SchemaVersion int64            `json:"schema_version"`
}
```

### Check List 

Tests 

Code changes

 - Has exported function/method change

Side effects

Related changes

---
This change is [Reviewable](https://reviewable.io/reviews/pingcap/tidb/8256)

### CODE REVIEW

**Comment by crazycs520** [comment]
@zanmato1984 PTAL

File: server/http_handler.go [hunk 1]
```diff
@@ -1362,3 +1367,45 @@ func (h allServerInfoHandler) ServeHTTP(w http.ResponseWriter, req *http.Request
...
+	if err != nil {
+		writeError(w, err)
+		return
+	}
+
+	dbTblInfo := DBTableInfo{
+		SchemaVersion: schema.SchemaMetaVersion(),
+	}
+	tbl, ok := schema.TableByID(int64(tblID))
+	if !ok {
```
Comment (zanmato1984) [inline server/http_handler.go]: I imagine that this `!ok` means table with ID `tblID` doesn't exist, right? If so, how about returning an error like "Table which ID = %s does not exist", just as how `schemaHandler` did for a non-existing table ID?

I think it doesn't make much sense that user gets a json only containing a schema version for a non-existing table when calling this API.

File: server/http_handler.go [hunk 2]
```diff
@@ -1362,3 +1367,45 @@ func (h allServerInfoHandler) ServeHTTP(w http.ResponseWriter, req *http.Request
 	}
 	writeData(w, clusterInfo)
 }
+
+// DBTableInfo is use ro report the db, table info and the current schema version.
```
Comment (zanmato1984) [inline server/http_handler.go]: `use ro` -> `used to`.

File: server/http_handler.go [hunk 3]
```diff
@@ -1362,3 +1367,45 @@ func (h allServerInfoHandler) ServeHTTP(w http.ResponseWriter, req *http.Request
...
+type DBTableInfo struct {
+	DBInfo        *model.DBInfo    `json:"db_info"`
+	TableInfo     *model.TableInfo `json:"table_info"`
+	SchemaVersion int64            `json:"schema_version"`
+}
+
+//ServeHTTP handles request of db info and table info that related to the  tableID.
```
Comment (zanmato1984) [inline server/http_handler.go]: Missing space after leading `//`, and redundant space before tailing `tableID`.

**Comment by zanmato1984** [comment]
LGTM. I think you might need another approve from some TiDB team member?

**Comment by crazycs520** [comment]
@winkyao @ciscoxll PTAL

**Comment by jackysp** [comment]
I think we could get the table info by
https://github.com/pingcap/tidb/blob/f497444389fbf8da58ffe53552e06bfcb07692d8/docs/tidb_http_api.md#L52
So could we append some DB info to the old API?

File: server/http_handler_test.go [hunk 4]
```diff
@@ -535,11 +535,11 @@ func (ts *HTTPHandlerTestSuite) TestGetSchema(c *C) {
 
 	resp, err = http.Get(fmt.Sprintf("http://127.0.0.1:10090/schema?table_id=5"))
 	c.Assert(err, IsNil)
-	var t *model.TableInfo
+	var dbTable *DBTableInfo
 	decoder = json.NewDecoder(resp.Body)
-	err = decoder.Decode(&t)
+	err = decoder.Decode(&dbTable)
 	c.Assert(err, IsNil)
-	c.Assert(t.Name.L, Equals, "user")
```
Comment (zimulala) [inline server/http_handler_test.go]: Add tests for schema version?

File: server/http_handler.go [hunk 5]
```diff
@@ -585,6 +586,13 @@ func (h binlogRecover) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	binloginfo.DisableSkipBinlogFlag()
 }
 
+// DBTableInfo is use to report the db, table info and the current schema version.
```
Comment (zimulala) [inline server/http_handler.go]: s/use/used
s/db/DB

**Comment by crazycs520** [comment]
@winkyao PTAL

**Pull request merged**